indico/modules/designer/templates/template.html

```diff
@@ -14,6 +16,38 @@
 {% endblock %}
 
 {% block content %}
+    {% if tpls_count %}
+        <div class="affected-targets-warning weak-hidden">
+            {% call message_box('warning', fixed_width=true) %}
+                <p>
+                    {% set link_start %}<a href="#" class="js-affected-objects">{% endset %}
+                    {% set link_end %}</a>{% endset %}
+                    {%- trans count=tpls_count, link_start=link_start|safe, link_end=link_end|safe -%}
```

Comment:
[set blocks are safe by default](https://github.com/pallets/jinja/commit/6a754cb83a6686534dbc088f677f7a1e7ca3971e), so you can get rid of the whole `, link_start=link_start|safe, link_end=link_end|safe` part